README.rst

```diff
@@ -36,11 +36,27 @@ Usage
     # ...and into which we install our project and its testing requirements.
     $ env/bin/pip install -e ".[testing]"
 
-3. Configure the database.
+3. Initialize the database.
 
 .. code-block:: bash
 
-    $ env/bin/initialize_myproj_db development.ini
+    # This cookiecutter provides support for initializing the database either
+    # (a) with migrations through alembic, or
+    # (b) without migrations.
+    #
+    # (a) Initialize the database with migrations through alembic:
+    # First use alembic to generate revisions.
+    $ env/bin/alembic -c development.ini revision --autogenerate -m "init"
+    # upgrade to that revision:
+    $ env/bin/alembic -c development.ini upgrade head
+    # Later you can run database migrations.
+    # Don't forget `-c development.ini` option.
```

Comment:
This comment for "Don't forget..." is redundant to the commands above it. Please remove for all instructions.